launcher/ui/pages/modplatform/flame/FlameModPage.cpp

```diff
@@ -38,6 +38,7 @@
 
 #include "FlameModModel.h"
 #include "ui/dialogs/ModDownloadDialog.h"
+#include <qglobal.h>
```

Comment:
This import is not needed (nvim moment? :^) )

Reply:
Yes nvim moment